src/app/tests/crud/test_crud_robot_type.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.robot_type import RobotType
+from app.crud.crud_robot_type import robot_type
+from app.schemas.robot_type import RobotTypeUpdate
+from app.tests.utils.models import create_robot_type
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_robot_type_create(database: AsyncSession):
+    robot_type_obj = await create_robot_type(db=database)
+    assert robot_type_obj is not None
+    assert isinstance(robot_type_obj, RobotType)
```

Comment:
Please also check the that the object has arrived in the database. (And not just that I return the request body and still get a pass :wink:)

Reply:
Addressed